androidApp/src/main/java/com/mbta/tid/mbta_app/android/stopDetails/TripStops.kt

```diff
@@ -101,7 +102,16 @@ fun TripStops(
     Box {
         Box(
             Modifier.matchParentSize()
-                .padding(4.dp)
+                .padding(horizontal = 4.dp)
+                .padding(
+                    bottom =
+                        if (
+                            stops.stops.lastOrNull()?.disruption?.alert?.significance ==
+                                AlertSignificance.Major
+                        )
+                            4.dp
+                        else 0.dp
+                )
```

Comment:
Including padding at the bottom for the background makes sense if the list ends with an alert card, but does not make sense if the list ends with a stop with no alert card.